docs: add metadata to all remaining pages (#1246)
canonical/rockcraft: docs/explanation/pebble.rst

Reviewer: Something went wonky with a carriage return on this page (from a Windows machine?), causing the whole diff to go haywire. Can you take a look?

Author: Fixed it!

.. meta::

>     :description: Overview of Pebble, the default service manager and entrypoint for rocks. Pebble orchestrates container processes and service layers.

Reviewer: ```suggestion
    :description: Overview of Pebble, the default container entrypoint for rocks. Pebble acts as a service manager, and can handle multiple processes inside the rock at once.

```

>     :description: Explanation of Pebble, the default service manager and entrypoint for all Rockcraft rocks. Learn how Pebble orchestrates container processes and service layers.

Reviewer: Food for thought:

```suggestion
    :description: Overview of Pebble, the default service manager and entrypoint for rocks. Pebble orchestrates container processes and service layers.
```

- The guidelines aren't meant to be utterly strict, so you don't have to start every explanation the same way. That a product could be "explained" doesn't sound right to me (consider someone saying "Explain Ubuntu 26.04 to me"), so my suggestion might be more suitable.
- I suggest removing "learn how" so we can reserve that verb for tutorials.
